src/EditorFeatures/CSharpTest/CodeActions/ImplementInterface/ImplementImplicitlyTests.cs

```diff
@@ -266,8 +266,7 @@ public readonly void Goo1() { }
     [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/70232")]
     public async Task TestMissingWhenAlreadyContainingImpl()
     {
-        await TestMissingAsync(
-            """
+        await TestInRegularAndScriptAsync("""
```

Comment:
>TestInRegularAndScriptAsync [](http://example.com/codeflow?start=14&length=27)

nit: test name should probably change

Reply:
Good point. Well fix in follow-up. The original tray was just validating no crash. The actual behavior is fine either way.